ex-1/define.h

```diff
@@ -0,0 +1,14 @@
+
+
+#define STRING_LENGTH 300
+#define STRING_WORDS_NUMBER 30
+#define WORD_LENGTH 10
+
+
+int sumOdd(unsigned int);
```

Comment:
Please make the function names consistently start with upper case letter. For ex: SumOdd.

Reply:
OK anh.